Cargo.toml

```diff
@@ -34,14 +34,22 @@ moka = { version = "0.12", features = ["future"] }
 num = "0.4"
 prometheus = "0.13"
 prometheus-metric-storage = "0.5.0"
-reqwest = "0.11"
+reqwest = { version = "0.11", default-features = false, features = [
+    "json",
+    "rustls-tls",
+] }
 serde = "1"
 serde_json = "1"
 serde_with = "3"
```

Comment:
Getting a build failure from the `app-data` crate:

```
error[E0432]: unresolved import `serde_with::serde`
  --> app-data/src/app_data_hash.rs:26:17
   |
26 |     serde_with::serde::{Deserialize, Serialize},
   |                 ^^^^^ could not find `serde` in `serde_with`
```

Looks like [app_data_hash.rs](cci:7://file:///Users/aryan/.cargo/git/checkouts/services-dec19295c16c25a8/6c521a8/crates/app-data/src/app_data_hash.rs:0:0-0:0) imports `serde_with::serde::{Deserialize, Serialize}`, but `serde_with` v3 doesn't have that re-export anymore (it was removed in the v2 → v3 upgrade). Not sure how this is working on main - maybe I'm missing something in my setup?

Reply:
Why is the PR updating the dependencies in the first place? Unless a PR needs a dependency bump because it makes use of a new lib or features let's separate maintenance dependency bumps from new code.